feat: support new data expiration rule attribute in online archive (#1528)
mongodb/terraform-provider-mongodbatlas: website/docs/d/online_archives.html.markdown

# Online Archive
## Attributes reference
* `db_name` - Name of the database that contains the collection.
* `coll_name` -  Name of the collection.

> * `collection_type` - Classification of MongoDB database collection that you want to return, "TIMESERIES" or "STANDARD". Default is "STANDARD". 

Reviewer: ```suggestion
* `collection_type` - Type of MongoDB collection that you want to return. This value can be "TIMESERIES" or "STANDARD". Default is "STANDARD". 
```
